### ISSUE TITLE
[BUG] resizePath always have null value

### ISSUE THREAD

**Issue by dhruvinsh** [immich-app/immich#563]
**Describe the bug**
Upon uploading the image/picture on the webpage, thumbnail never gets generated. From the database I can see that resizePath always stay null.

I am using docker to setup all the services, I am went to each container to see if upload folder is writable or not and it is.
Thubmnial call returns below response: http://192.168.2.62:2283/api/asset/thumbnail/43f3e3bb-256a-4761-8207-71f4f2784d78?format=JPEG
```json
{
  "response": {
    "statusCode": 404,
    "message": "resizePath not set",
    "error": "Not Found"
  },
    "status": 404,
    "message": "resizePath not set",
    "name": "NotFoundException"
}
```
**Task List**

*Please complete the task list below. We need this information to help us reproduce the bug or point out problems in your setup. You are not providing enough info may delay our effort to help you.*

- [✅] I have read thoroughly the README setup and installation instructions.
- [❌] I have included my `docker-compose` file. using unraid. but strictly following [docker-compose](https://raw.githubusercontent.com/immich-app/immich/main/docker/docker-compose.yml)
- [❌] I have included my redacted `.env` file. using this [.env file](https://raw.githubusercontent.com/immich-app/immich/main/docker/.env.example)
- [✅] I have included information on my machine, and environment.

**Expected behavior**
Based on a code ```const resizePath = `upload/${asset.userId}/thumb/${asset.deviceId}/`;``` based on userID and deviceId a folder should be created but never happens!

**System**
- Server Version: `v1.26.0`

**Additional context**
Using webpage only to upload the picture.

**Comment by alextran1502** [comment]
Can you provide more information how do you get it up and running in Unraid? If you are not following docker-compose, most likely that is the trouble spot

**Comment by dhruvinsh** [comment]
@alextran1502 Thanks for pointing it out. Problem is solved, it was because of redis

I have two redis version v6 and v6.2 (immich needs 6.2)

Intially I started with v6 only and did not see thumbnail, turns out microservice docker was missing the redis config. I added afterwards but v6.2 on microservice and started the container. Then started chanaging other container to v6.2 as well still no chnage.

Solution: I had to delete the postgres database and start from scratch and it is wokring fine now.
